docstore/doc.go

```diff
@@ -111,11 +112,15 @@
 //    type Book struct {
 //       Title           string `docstore:"title"`
 //       Author          Name   `docstore:"author"`
-//       PublicationYear int    `docstore:"pub_year,omitempty"`
+//       PublicationYears []int `docstore:"pub_years,omitempty"`
+//       // Title in lower case.
+//       LowerTitle      string `docstore:"-"`
 //    }
 //
-// This struct describes a document with field names "title", "author" and "pub_year".
-// The pub_year field is omitted from the stored document if it is zero.
+// This struct describes a document with field names "title", "author" and
+// "pub_years". The pub_years field is omitted from the stored document if it has
+// length zero. The LowerTitle field is never stored, because it can easily be
+// computed from the Title field.
```

Comment:
How about number of publications for something that can be computed from another field?

Reply:
Done.